share/graphql/query.py

```diff
@@ -41,7 +41,7 @@ def resolve_me(self, args, context, info):
     @graphene.resolve_only_args
     def resolve_sources(self, limit=25, offset=0):
         # TODO Long title will need an index
-        return models.ShareUser.objects.exclude(long_title='').exclude(favicon='').order_by('long_title')[offset:offset + limit]
+        return models.ShareUser.objects.exclude(long_title='').exclude(icon='').order_by('long_title')[offset:offset + limit]
```

Comment:
Looks like `long_title` doesn't exist any more and chaining seems to result in an `AND`

```python
In [6]: str(Source.objects.exclude(long_title='').exclude(icon='').query)
Out[6]: 'SELECT "share_source"."id", "share_source"."name", "share_source"."long_title", "share_source"."home_page", "share_source"."icon", "share_source"."user_id" FROM "share_source" WHERE (NOT ("share_source"."long_title" = ) AND NOT ("share_source"."icon" =  AND "share_source"."icon" IS NOT NULL))'
```